### ISSUE TITLE
New Feature: User/Community Report Confirmation

### ISSUE THREAD

**Issue by rsbaher1** [AnjaSlama/Pedestrian-Safety-Reporting#4]
When an issue is reported, users should be able to confirm if the issue is still there or not, similar to how Google Maps/Wayze asks users to confirm if a previously reported Police or hazard is still there.

**Comment by AnjaSlama** [comment]
@rsbaher1  I agree. It is a good idea to collect this information from the users.
